src/test/java/uk/gov/hmcts/ccd/domain/service/casefileview/CategoriesAndDocumentsServiceTest.java

```diff
@@ -111,7 +112,10 @@ void testShouldTransformAll() {
     void testShouldResolveDocumentCategory(final String categoryOnDocument,
```

Comment:
Please add more unit test to cover all possible conditions for the results of **underTest.resolveDocumentCategory**